<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>610</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">*
{
font-family:century gothic;
font-size:14px;
}

QFrame {	
	background-color: rgb(56, 58, 89);	
	color: rgb(220, 220, 220);
	border-radius: 10px;
}

QPushButton
{
background-color: qlineargradient(spread:pad, x1:0, y1:0.511364, x2:1, y2:0.523, stop:0 rgba(254, 121, 199, 255), stop:1 rgba(170, 85, 255, 255));
border-radius:60px;

}
QToolButton
{
background-color: qlineargradient(spread:pad, x1:0, y1:0.511364, x2:1, y2:0.523, stop:0 rgba(254, 121, 199, 255), stop:1 rgba(170, 85, 255, 255));
border-radius:60px;

}
QLabel
{
color:white;
background:transparent;
}

QPushButton
{
color:white;
border-radius:15px;
}

QPushButton:hover
{
color:#333;
border-radius:15px;
background:#49ebff;
}

QLineEdit
{
font-size:18px;
border:2px solid rgb(45, 46, 71);
border-radius:20px;
color:#FFF;
padding-left:20px;
padding-right:20px;
background-color:rgb(45, 46, 71);
}
QLineEdit:hover
{
border:2px solid rgb(48,50,62);
}

QLineEdit:focus{
border: 2px solid rgb(85,170,255);
background-color:rgb(43,45,56);
}


</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>30</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Time</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="resultID_Edit_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>330</y>
        <width>159</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>410</y>
        <width>67</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Accuracy</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="resultname_Edit">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>330</y>
        <width>159</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="resultID_Edit">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>290</y>
        <width>159</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="resultrace_Edit">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>370</y>
        <width>159</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="resultlabel_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>201</width>
        <height>201</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>34</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Date</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="resultID_Edit_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>290</y>
        <width>159</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="accuracy_Edit">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>410</y>
        <width>159</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="resultlabel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>201</width>
        <height>201</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>330</y>
        <width>42</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>290</y>
        <width>16</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>370</y>
        <width>37</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Race</string>
      </property>
     </widget>
     <widget class="QFrame" name="title_bar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>681</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: none;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_title_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>112</width>
         <height>50</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>century gothic</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>15</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
      <widget class="QFrame" name="frame_btns_3">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>0</y>
         <width>100</width>
         <height>31</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="btn_minimize_3">
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>17</width>
            <height>17</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Minimize</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	border: none;
	border-radius: 8px;		
	background-color: rgb(255, 170, 0);
}
QPushButton:hover {	
	background-color: rgba(255, 170, 0, 150);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_close_3">
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>17</width>
            <height>17</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Close</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	border: none;
	border-radius: 8px;		
	background-color: rgb(255, 0, 0);
}
QPushButton:hover {		
	background-color: rgba(255, 0, 0, 150);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_title_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>-10</y>
         <width>171</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>century gothic</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size:18px;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Face Result</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
